js/events/utils/index.js

```diff
@@ -148,10 +148,11 @@ export const needSkipEvent = e => {
     // TODO: this checking used in swipeable first move handler. is it correct?
     const { target } = e;
     const $target = $(target);
+    const isDropDown = $target.is('.dx-dropdownlist-popup-wrapper *, .dx-dropdownlist-popup-wrapper');
```

Comment:
It lloks like the coupling becomes too height here. Why is it? Is it possible to avoid event utils to know about DropDown concept?

Reply:
We haven't found good and safe solution to fix this inside DropDown files. Desided that 1 more comparsion in event/utils is not so bad